drafts/use-cases/7.searching-events.md

```diff
@@ -0,0 +1,120 @@
+﻿## Searching for events
+
+### Story
+As an application user
+I want to be able to quickly search through my events
+So I find events I'm interested in.
+
+As an API consumer
+I want to be able to search through resources
```

Comment:
Should we make this "through a collection of resources"? I don't think we have a good story about search in general yet